API.md

```diff
@@ -38,6 +38,8 @@ of iD (e.g. `https://ideditor-release.netlify.app`), the following parameters ar
   _Example:_ `map=20.00/38.90085/-77.02271`
 * __`maprules`__ - A path to a [MapRules](https://github.com/radiant-maxar/maprules) service endpoint for enhanced tag validation.<br/>
   _Example:_ `maprules=https://path/to/file.json`
+* __`notes`__ - Enables the notes layer by default.<br/>
+  _Example:_ `notes=1`
```

Comment:
```suggestion
  _Example:_ `notes=true`
```

I suggest to change this to `true` to be in line with `walkthrough=true` and `rtl=true`

Reply:
sure, updated